### PR TITLE
Added check for the processes in the backend server

### PR DESCRIPTION
**Pull request by eoliveros**
I changed teh base to ansible. Hopefully its correct.

### CODE REVIEW

File: provisioning/templates/telegraf_processes.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+               "/opt/telegraf/telegraf_processes.sh redis",
+               "/opt/telegraf/telegraf_processes.sh redis-sentinel",
+               "/opt/telegraf/telegraf_processes.sh alertcenter",
+               "/opt/telegraf/telegraf_processes.sh alertcenter_mailer",
+               "/opt/telegraf/telegraf_processes.sh marketprice",
+               "/opt/telegraf/telegraf_processes.sh matchengine",
+               "/opt/telegraf/telegraf_processes.sh readhistory",
+               "/opt/telegraf/telegraf_processes.sh accesshttp",
+               "/opt/telegraf/telegraf_processes.sh accessws",
+               "/opt/telegraf/telegraf_processes.sh nginx"
```
Comment (djpnewton) [inline provisioning/templates/telegraf_processes.conf.j2]: there is no nginx on the backend..

**Comment by eoliveros** [comment]
It looks like there is:

root@backend:~# ps aux |grep nginx
root      7388  0.0  0.0 124980  1352 ?        Ss   Jul03   0:00 nginx: master process /usr/sbin/nginx -g daemon on; master_process on;
www-data  7389  0.0  0.0 125356  3848 ?        S    Jul03   3:01 nginx: worker process
www-data  7390  0.0  0.0 125356  2308 ?        S    Jul03   3:05 nginx: worker process
root     20543  0.0  0.0  12944   972 pts/0    S+   02:19   0:00 grep --color=auto nginx
root@backend:~#

Also I remember reading somewhere that you need it for either the accesshttp or accessws. But could be wrong.

Erick

**Comment by djpnewton** [comment]
It looks like nginx was accidentally left in (https://github.com/djpnewton/viabtc_exchange_server/blob/ansible/provisioning/deploy.yml#L18), but we dont configure it anywhere

**Comment by eoliveros** [comment]
Ok, will delete it and push it again after I make some corrections on the others.


Erick

**Pull request merged**